_rules/95e043.md

```diff
@@ -0,0 +1,202 @@
+---
+name: Auto-play audio has a control mechanism
+test_type: atomic
+
+description: |
+  This rule checks that each auto-play audio or video element has a control mechanism.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>` `<video>` or `<source>` `src` file content with a `duration` of more than 3 seconds, that is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
+
+## Expectation 1
+
+For each test target a control mechanism is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The control mechanism to pause or stop or turn the audio volume off is [visible](#visible), has an [accessible name](#accessible-name) and is one of the first five focusable elements in the [document](https://www.w3.org/TR/dom/#concept-document).
```

Comment:
Why should it have to be one of the first five focusable elements? I know there is a technique that tells you to place this option near the top of the page, but I don't see this in the SC itself.

Reply:
Thats a fair point @ShadowBB. I will remove it from the **Expectation** and add a link under **Background**.